test/functions/lambda/lambdaThrows.chpl

```diff
@@ -1,8 +0,0 @@
-const f1 = lambda(n: int): int throws { return n*n; };
-const f2 = lambda(n: int): int throws { throw new Error(n:string); };
```

Comment:
Can the new Fcfs throw?

Reply:
Yes they can, and there is a test for that in `functions/fcf/Motivators.chpl` that mirrors this one.